### PR TITLE
feat(content): fedora 34

### PR DESCRIPTION
**Pull request by tim-rackn**
fedora 33 and 34 do not include fipscheck.
fedora 34 requires new boot options and kickstart.
Verified boot and install for f31, 33 and 34 on both legacy and UEFI.

### CODE REVIEW

File: content/params/linux-install-bootenv-map.yaml [hunk 1]
```diff
@@ -29,8 +29,9 @@ Schema:
     "debian-8": "debian-8-install"
     "debian-9": "debian-9-install"
     "debian-10": "debian-10-install"
-    "fedora-latest": "fedora-31-install"
-    "fedora-31": "fedora-31-install"
+    "fedora-latest": "fedora-34-install"
```
Comment (galthaus) [inline content/params/linux-install-bootenv-map.yaml:32]: Please don't delete fedora-31-install from the table.  Someone might be using it...

File: content/params/linux-install-bootenv.yaml [hunk 2]
```diff
@@ -31,7 +31,8 @@ Schema:
     - "debian-9"
     - "debian-10"
     - "fedora-latest"
-    - "fedora-31"
```
Comment (galthaus) [inline content/params/linux-install-bootenv.yaml:34]: Same here.

Comment (tim-rackn) [reply content/params/linux-install-bootenv.yaml:34]: Thanks for catching this.  I thought I left it in there.

File: content/params/linux-install-bootenv-map.yaml (hunk 1 above)
Comment (tim-rackn) [reply content/params/linux-install-bootenv-map.yaml:32]: Let me look at this again.  I thought I just bumped latest to fedora-34, but left fedora-31 in there.

Comment (tim-rackn) [reply content/params/linux-install-bootenv-map.yaml:32]: Fixed.

File: content/params/linux-install-bootenv.yaml (hunk 2 above)
Comment (tim-rackn) [reply content/params/linux-install-bootenv.yaml:34]: Fixed.

**Pull request merged**